lib/tasks/gettext.rake

```diff
@@ -90,4 +90,26 @@ namespace :gettext do
       end
     end
   end
+
+  desc "Extract human locale names from translation catalogs and store them in a yaml file"
+  task :extract_locale_names do
```

Comment:
We're in the gettext namespace, we might want to move our stuff to a namespace we control in a future PR